src/encoded/types/dataset.py

```diff
@@ -213,6 +214,170 @@ def assembly(self, request, original_files, status):
     def hub(self, request):
         return request.resource_path(self, '@@hub', 'hub.txt')
 
+    @calculated_property(condition='analysis_objects', schema={
+        "title": "Default analysis",
+        "description": "One default analysis that should be checked first.",
+        "comment": "Do not submit. This field is calculated from files in this analysis.",
+        "type": "string",
+        "linkTo": "Analysis",
+        "notSubmittable": True
+    })
+    def default_analysis(self, request, status, analysis_objects):
+        unreleased_status = ['in progress', 'submitted', 'deleted', 'replaced']
+        status_order = load_schema(
+            'encoded:schemas/analysis.json'
+        )['properties']['status']['enum']
+        if status not in unreleased_status:
+            status_order = [
+                s for s in status_order if s not in unreleased_status
+            ]
+        award_rfa_order = load_schema(
```

Comment:
I don't think we should use raw `load_schema` calls here (having just looked at this stuff for abstract facets) since it's going to read from disk every time. You should be able to access schemas through the request object with something like `request.registry[TYPES]['award'].schema`.

Reply:
replaced load_scema following @keenangraham suggestion